basics/getting_started.md

```diff
@@ -3,18 +3,22 @@ layout: contribute
 title: Getting Started
 ---
 
-# Setting up your coding environment
+# Getting started
 
-For now we have support for IntelliJ. We don't have IDE support for other
-languages in Bazel right now.
+This page helps you set up your coding environment, describes creating an
+IntelliJ project and takes you through compiling and debugging your project.
+Finally, you can read about Bazel's source code, how to navigate through it, and
+how to use the Bazel CI system.
 
-## Preparations
+## Installing Bazel
 
 *  [Install Bazel](https://bazel.build/versions/master/docs/install.html) on your
```

Comment:
These are instructions, so they should be ordered lists, not bulleted.

Reply:
Done.